selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/AddOrImportProjectFormTest.java

```diff
@@ -412,14 +410,14 @@ public void checkProjectsBlank() throws Exception {
   public void checkCreatingProject() throws Exception {
     // check that name field saves it state after choosing another stack
     newWorkspace.waitPageLoad();
-    newWorkspace.waitStackSelected(BLANK);
+    newWorkspace.waitStackSelected(Stack.BLANK);
```

Comment:
Please, restore static imports in this class.

Reply:
is there some specific reason the static imports are used? using static imports should be avoided.

Reply:
Why it should be avoided? In this case it makes the code clearer because *Stack.* prefix doesn't add significant info to the code.

You just need to restore initial content of file.

Reply:
https://carlosbecker.com/posts/avoid-static-imports/
Here's a nice article explaining the issues that may arise, when developers that have never seen the code try to contribute.
Still it's just my opinion. I can restore the class.

Reply:
I agree that static imports can arise the problems, and they are actually not recommended to use in production. But here we have tests, and also there are situations when static imports make code much more readable, for example, in lambdas.
Anyway, these changes don't block merging into master, IMHO.